src/phylum/ci/cli.py

```diff
@@ -61,6 +62,63 @@ def detect_ci_platform(args: argparse.Namespace, remainder: List[str]) -> CIBase
     return ci_env
 
 
+def ensure_project(ci_env: CIBase) -> None:
+    """Ensure a Phylum project is created and in place.
+
+    When a project is specified through arguments, attempt to create that project,
+    overwriting any `.phylum_project` file that already exists.
+    Continue on without error when a specified project already exists.
+    When a group is specified through arguments, attempt to use that group.
+    """
+    if not ci_env.phylum_project:
+        return
+
+    cmd = f"{ci_env.cli_path} project create {ci_env.phylum_project}"
+    if ci_env.phylum_group:
+        ensure_group(ci_env)
+        print(f" [-] Using provided Phylum group: {ci_env.phylum_group}")
+        cmd = f"{ci_env.cli_path} project create --group {ci_env.phylum_group} {ci_env.phylum_project}"
+
+    print(f" [*] Creating a Phylum project with the name: {ci_env.phylum_project} ...")
+    if ci_env.phylum_project_file.exists():
+        print(f" [+] Overwriting existing `.phylum_project` file found at: {ci_env.phylum_project_file}")
+    try:
+        # The Phylum CLI will return a unique error code of 14 when a project that already exists is attempted to be
+        # created. This situation is recognized and allowed to happen since it means the project exists as expected.
+        # Any other exit code is an error and a reason to re-raise.
+        subprocess.run(shlex.split(cmd), check=True)
+    except subprocess.CalledProcessError as err:
+        if err.returncode == 14:
+            print(f" [-] Project {ci_env.phylum_project} already exists. Continuing with it ...")
+            return
+        print(f" [!] There was a problem creating the project with command: {cmd}")
+        raise
```

Comment:
If you set `check=False`, you can avoid the exception and (IMO) clean up the code a bit.

```suggestion
    ret = subprocess.run(shlex.split(cmd), check=False)
    # The Phylum CLI will return a unique error code of 14 when a project that already exists is attempted to be
    # created. This situation is recognized and allowed to happen since it means the project exists as expected.
    # Any other exit code is an error.
    if ret.returncode == 0:
        print(f" [-] Project {ci_env.phylum_project} created successfully.")
    elif ret.returncode == 14:
        print(f" [-] Project {ci_env.phylum_project} already exists. Continuing with it ...")
    else:
        ret.check_returncode()
```